### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by aymanhab**
Add entries to changelog corresponding to IMU component and IMUDataReporter.

Fixes issue #<issue_number>

### CHANGELOG.md (choose one)

- no need to update because...
- updated.

---
This change is [Reviewable](https://reviewable.io/reviews/opensim-org/opensim-core/3069)

### CODE REVIEW

**Comment by aymanhab** [comment]
@carmichaelong to review wording, thank you.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ This is not a comprehensive list of changes but rather a hand-curated collection
 
 v4.3
 ====
+- Introduced IMU component that models industrial Inertial Measurement Units (IMUs) with corresponding output signals. 
+- Introduced IMUDataReporter (analysis) to record signals from IMUs components placed on models.
```
Comment (carmichaelong) [inline CHANGELOG.md:12]: "IMUs components" -> "IMU components"

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,8 @@ This is not a comprehensive list of changes but rather a hand-curated collection
 
 v4.3
 ====
+- Introduced IMU component that models industrial Inertial Measurement Units (IMUs) with corresponding output signals.
```
Comment (carmichaelong) [inline CHANGELOG.md:11]: - Remove "industrial"
- Could add ".. with corresponding output signals, including orientation, accelerometer, and gyroscope signals"

**Review by carmichaelong — COMMENTED** [review]
Thanks @aymanhab. Just a few comments on wording

**Review by aymanhab — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/opensim-org/opensim-core/3069)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @aymanhab and @carmichaelong)

___
*[CHANGELOG.md, line 12 at r1](https://reviewable.io/reviews/opensim-org/opensim-core/3069#-MhpSVWMtUq8du3HMVhI-r1-12:-Mhq3UH5AsfKa9_z74RH:b-896fix) ([raw file](https://github.com/opensim-org/opensim-core/blob/26d0adda03f9363ae929a3243b66cd591109e8fc/CHANGELOG.md#L12)):*
<details><summary><i>Previously, carmichaelong wrote…</i></summary><blockquote>

"IMUs components" -> "IMU components"
</blockquote></details>

Done.

**Review by aymanhab — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/opensim-org/opensim-core/3069)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @carmichaelong)

___
*[CHANGELOG.md, line 11 at r1](https://reviewable.io/reviews/opensim-org/opensim-core/3069#-MhpSVWMtUq8du3HMVhI-r1-11:-Mhq3hDK4P97QQeGDxfR:b-896fix) ([raw file](https://github.com/opensim-org/opensim-core/blob/26d0adda03f9363ae929a3243b66cd591109e8fc/CHANGELOG.md#L11)):*
<details><summary><i>Previously, carmichaelong wrote…</i></summary><blockquote>

- Remove "industrial"
- Could add ".. with corresponding output signals, including orientation, accelerometer, and gyroscope signals"
</blockquote></details>

Done.

**Review by carmichaelong — APPROVED** [review]

**Pull request merged**